### ISSUE TITLE
No timestamps in logs

### ISSUE THREAD

**Issue by sheerun** [heketi/heketi#694]
It's harder to debug this way. Could you include some kind of timestamps in the logs?

**Comment by lpabon** [comment]
Is this not ok `[heketi] ERROR 2017/03/01 04:14:32` ?

**Comment by sheerun** [comment]
it is, but not all of the services have this timestamp. Notably [negroni] is missing them as there are many logs from it. There are some outliers like "Listening on port 8080", "Copied backup db to /var/lib/heketi/heketi.db", "Heketi v4.0.0-21-g50344fc"

A consistent log format with timestamps everywhere would be perfect :)

**Comment by phlogistonjohn** [comment]
I was looking for old bugs to close when I noticed this and initially thought this was something I could close. But when I checked a heketi instance running from heketi:dev docker hub image it lacked the timestamps. However, my local build has them. This implies that either something has changed between negroni versions (and the binary was built using different versions) or something environmental.

Anyway, I agree that having timestamps would be preferred, and its possible, we just need to figure out how to make it work consistently. :-)
